File: Programmers/예산.py
def solution(d, budget):
    d.sort()
    answer = 0
    part_idx = 0
    while budget > 0 and part_idx < len(d):
        if budget >= d[part_idx] :
            answer += 1
            budget -= d[part_idx]
            part_idx += 1
        else:
            break;
    return answer

File: Programmers JS/예산.js
function solution(d, budget) {
  var answer = 0;
  d = d.sort((a, b) => a - b);
  let i = 0;
  while (true) {
    if (budget - d[i] >= 0) {
      budget -= d[i];
      i++;
      answer++;
    } else {
      break;
    }
  }
  return answer;
}
